File: src/main/java/io/github/marcuscastelo/invtweaks/util/KeyUtils.java
package io.github.marcuscastelo.invtweaks.util;

import net.minecraft.client.MinecraftClient;
import net.minecraft.client.util.InputUtil;

public class KeyUtils {
    public static boolean isKeyPressed(int key) {
        return InputUtil.isKeyPressed(MinecraftClient.getInstance().getWindow().getHandle(), key);
    }
}

File: src/main/kotlin/io/github/marcuscastelo/invtweaks/util/KeyUtils.kt
package io.github.marcuscastelo.invtweaks.util

import net.minecraft.client.MinecraftClient
import net.minecraft.client.util.InputUtil

object KeyUtils {
    @JvmStatic
    fun isKeyPressed(key: Int): Boolean {
        return InputUtil.isKeyPressed(MinecraftClient.getInstance().window.handle, key)
    }
}